### DISCUSSION TITLE
SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it

### DISCUSSION THREAD

**Discussion by AriscaPramalistyo05** [leokhoa/laragon#784] [Q&A]
![Cuplikan layar 2024-05-18 203036](https://github.com/leokhoa/laragon/assets/127287422/a1cee208-f0f9-4ee1-95de-48f6d912d982)
in the case eror from mysql but i run in the another project it's work but not in this project

**Comment by Pen-y-Fan** [comment]
Check you have fully installed the project, check:

- the **.env** file has the correct db settings
- check the **vendor** folder, run **composer install**
- Check MySQL is started: from the command line run `mysql -u root`, if you get the MySQL prompt `exit`

**Comment by Hamed042gh** [comment]
turn off fire wall then check...

**Reply by RodneySsemambo** [reply]
how

**Reply by Hamed042gh** [reply]
windows defender
